lib/docbuild.js

```diff
@@ -0,0 +1,7 @@
+createDocs = require('node-offline-api').createDocs
+buildOptions = require('node-offline-api').buildOptions
```

Comment:
Thanks @AnshulMalik !

Wow, what in the world was I doing when I submitted that? I think I spent a little too much time with Ruby that day. :) 

I has been done! Using var for now, until the lowest version of Node is a specific, then I can cross-check the available ES6 sugar that can be safely used.